recipe/meta.yaml

```diff
@@ -1,29 +1,33 @@
+{% set name = "jupyter_bokeh" %}
 {% set version = "3.0.7" %}
 
 package:
-  name: jupyter_bokeh
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/j/jupyter_bokeh/jupyter_bokeh-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 2da8c3ddc734d15737bf06126d9e31e84d30f18ac3da3a3f95be40a95a054c87
 
 build:
-  number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --install-option="--skip-npm"
+  number: 1
```

Comment:
Why is this build number 1? I think it should be 0 as we have not uploaded jupyter_bokeh 3.0.7 to any channel yet?